src/Microsoft.Identity.Web/CertificateManagement/DefaultCertificateLoader.cs

```diff
@@ -116,6 +116,11 @@ private static X509Certificate2 LoadFromBase64Encoded(string certificateBase64,
 
             KeyVaultCertificateWithPolicy certificate = certificateClient.GetCertificate(certificateName);
 
+            if (certificate.Properties.NotBefore <= DateTime.UtcNow && certificate.Properties.ExpiresOn <= DateTime.UtcNow)
+            {
+                return null;
```

Comment:
There seem to be a mix of returning null and throwing exceptions. Would be interesting to try out the non-happy path of configuring MIW with some expired or bad certs and seeing the experience.

Reply:
Of course this was done, @bgavrilMS !
See https://github.com/AzureAD/microsoft-identity-web/issues/956#issuecomment-832760201 for all the pathological cases which were tested.